.github/workflows/build.yaml

```diff
@@ -0,0 +1,29 @@
+name: build
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version: [2.7, 2.6, 2.5]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ martix.ruby-version }}
```

Comment:
CI fails to run due to a typo: https://github.com/fluent/fluentd/actions/runs/515080827

```diff
- martix
* matrix
```